setup/Readme.md

```diff
@@ -94,7 +94,10 @@ wkz run
 
 ## Background
 
-Whenever you connect your Garmin device to your Raspberry Pi, workoutizer will automatically mount the device using
-`udev`. Since it is mounted as a [gvfs](https://en.wikipedia.org/wiki/GVfs) device, the file system of your device will
+Whenever you connect your Garmin device to your Raspberry Pi, workoutizer will automatically mount the device. Workoutizer currently supports devices with MTP and Block modes. Some devices support both modes, some only one. 
+
+The MTP devices are mounted using `udev`. Since it is mounted as a [gvfs](https://en.wikipedia.org/wiki/GVfs) device, the file system of your device will
```

Comment:
`udev` is used in both cases. `udev`-rules are triggered by system events and are used to post to the workoutizer api endpoint, which in turn mounts the device. Only after the endpoint was called, your implemented logic will  distinguish between MTP and Block devices and then uses either
* `gio` for mounting MTP devices, or
* `pmount` for mounting block devices.

Could you try to rephrase this sentence a bit to reflect this?

Reply:
I rephrased the sentence.